client/my-sites/plans-features-main/components/plan-upsell-modal/components/paid-domain-suggested-plan-section.tsx

```diff
@@ -23,6 +24,12 @@ export default function PaidDomainSuggestedPlanSection( props: {
 	const translate = useTranslate();
 	const { paidDomainName, onPlanSelected, isBusy } = props;
 
+	const previousCopy = translate( 'Free for one year. Includes Premium themes.' );
+	const updatedCopy = translate( 'Free for one year, includes Premium themes' );
+	const hasTranslationForUpdatedCopy = useHasEnTranslation()(
```

Comment:
This works fine, but I didn't give it a double take because it's not really idiomatic React. At least not in our code base. We'd usually call the hook near the top of the function (like `useTranslate`) even in this special case where it's only used once.

I'm not sure why we do it this way. Maybe because React has all these special rules for how hooks get called, and so sometimes doing it this way actually doesn't work, even when there's only one call to the function.

I searched the code the code and found there are actually other components doing it this way too, but we mostly don't.

You should just merge it as is, not worth another CI round trip.

Reply:
I was using this as an example, a starting point: https://github.com/Automattic/wp-calypso/blob/5855897201ecc35fa25f6a21f72346ea386559f0/client/me/account/toggle-sites-as-landing-page.tsx.

Reply:
Updated, now with idiomatic React. 😄 
Thanks for the input!